codegenerator/cli/templates/static/codegen/src/db/Migrations.res

```diff
@@ -118,6 +134,18 @@ let runUpMigrations = async (~shouldExit) => {
     | _ => ()
     }
 
+  let _ = await sql->unsafe(
+    `DO $$ 
+    BEGIN
+      ${reset ? `DROP SCHEMA IF EXISTS ${Env.Db.publicSchema} CASCADE;` : ``}
+      IF NOT EXISTS(SELECT 1 FROM information_schema.schemata WHERE schema_name = '${Env.Db.publicSchema}') THEN
+        CREATE SCHEMA ${Env.Db.publicSchema};
+        GRANT ALL ON SCHEMA ${Env.Db.publicSchema} TO postgres;
+        GRANT ALL ON SCHEMA ${Env.Db.publicSchema} TO public;
+      END IF;
+    END $$;`,
+  )
```

Comment:
Why does any of this run on the non reset case? I see it only drops in this case but what about the rest?

Reply:
For example, you run `pnpm start` with a different schema. In this case it won't trigger the down migration (since pnpm start doesn't have the logic), but it still should create the schema if it's missing, so it's able to work correctly.

Reply:
Ah got you 👍🏼